sample-app/src/components/UniversalResults.tsx

```diff
@@ -20,24 +30,31 @@ interface AppliedFiltersConfig extends Omit<DecoratedAppliedFiltersConfig, 'appl
 
 interface UniversalResultsProps {
   appliedFiltersConfig?: AppliedFiltersConfig,
-  verticalConfigs: Record<string, VerticalConfig>
+  verticalConfigs: Record<string, VerticalConfig>,
+  customCssClasses?: UniversalResultsCssClasses,
+  compositionMethod?: CompositionMethod
```

Comment:
nit: maybe the name for the filed in interface should be `cssCompositionMethod` to be a bit more clear